sylt-parser/src/statement.rs

```diff
@@ -671,6 +688,11 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
 
             if matches!(ctx.token(), T::External) {
```

Comment:
Consider making this a `match`? I might be nit picking too much though - I'm fine with the if-else.